lib/avalanche/requests/statement_request.ex

```diff
@@ -84,7 +84,10 @@ defmodule Avalanche.StatementRequest do
   end
 
   defp build_pipeline(request, opts) do
-    disable_polling = Keyword.fetch!(opts, :async)
+    async? = Keyword.fetch!(opts, :async)
+    streaming? = Keyword.get(opts, :streaming, false)
```

Comment:
hmm should this ALSO be `fetch!` because of how `NimbleOptions` will built it and take care of the default value I guess?